### PR TITLE
CAM-12946: doc(db): add postgresql 13 support

### PR DESCRIPTION
**Pull request by koevskinikola**
Related to CAM-12946

### CODE REVIEW

File: enterprise/content/announcement.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ __Changes in Supported Environments:__
 
 * Support for WildFly Application Server 21.0
 * Support for Amazon Aurora PostgreSQL compatible with PostgreSQL 10.13 (also supported as of 7.12.16+, 7.13.10+, 7.14.4+)
-* Support for PostgreSQL 10.13 (also supported as of 7.12.18+, 7.13.11+, 7.14.5+)
+* Support for PostgreSQL 10.13 (also supported as of 7.12.18+, 7.13.11+, 7.14.5+) / 13 (only for 7.15)
```
Comment (yanavasileva) [inline enterprise/content/announcement.md:29]: Small suggestion for better readability:
```suggestion
* Support for PostgreSQL 10.13 (also supported as of 7.12.18+, 7.13.11+, 7.14.5+) / PostgreSQL 13 (only for 7.15)
```

**Review by yanavasileva — APPROVED** [review]
Please have a look at my comment.

File: enterprise/content/announcement.md (hunk 1 above)
Comment (koevskinikola) [reply enterprise/content/announcement.md]: @yanavasileva you are right, the readability can be improved. However, I checked out your suggestion, and the line is too long with it.

What do you thing of the new changes?

Comment (yanavasileva) [reply enterprise/content/announcement.md]: Looks good

**Review by yanavasileva — APPROVED** [review]

**Pull request merged**